docs/llb/llb-api-spec.md

```diff
@@ -36,9 +38,46 @@ Any response of `401 UNAUTHORIZED` with the following body indicates that the us
   * [`PUT api/v1/protected/checkout/register/:event_id`](#-put-api-v1-protected-checkout-register--event-id-)
   * [`POST api/v1/webhooks/stripe`](#-post-api-v1-webhooks-stripe)
 
+# Getting Public Events
 
+## `GET api/v1/events?ids=1,2,3,...`
 
-# Getting Events
+Get the event bodies for the public events with the ids that are specified in the query parameter.
+
+### Query Params
+
+##### ids: INT-LIST
+
+The event ids that are being requested. Ids are numbers and they should be seperated by a single comma. If this query parameter is missing, this route returns an empty list. Ignore duplicated values. Ignore ids that do not correspond to any existing event.
```

Comment:
Wait this doesn't work, we won't have the IDs available on this frontend because we need this route to get them. So it should just return all of the events in the future when nothing is specified.